R/inboveg_classification.R

```diff
@@ -0,0 +1,51 @@
+#' Provide vegetation classifications
+#'
+#' Get the available vegatation classifications for the vegetation surveys
+#'
+#' @param connection odb DBIConnection-class, using DBI-ODBC like connectors
+#' @param survey_name vector of char or NULL (default NULL),
+#'     i.e. all surveys are taken into account
+#'
+#' @return data.frame
+#' @export
+#' @importFrom DBI dbSendQuery dbBind dbFetch dbClearResult
+#' @importFrom assertthat assert_that
+#'
+#' @examples
+#' \dontrun{
+#' library(DBI)
+#' con <- dbConnect(odbc::odbc(), dsn="Cydonia-prd")
+#' classifications <- inboveg_classification(con,
+#'                            survey_name = c('Sigma_LSVI_2012'))
+#' }
+inboveg_classification <- function(connection, survey_name = NULL) {
+
+    assert_that(is.vector(survey_name) | is.null(survey_name))
```

Comment:
an alternative. This requires no default for `survey_name`

```
if (missing(survey_name)) {
  survey_name <- NULL
} else {
  assert_that(is.character(survey_name))
}
```

Reply:
cool, thanks!